package.json

```diff
@@ -78,6 +78,7 @@
     "copy-dir": "0.3.0",
     "coveralls": "2.11.12",
     "deepcopy": "0.6.3",
+    "firefox-client": "0.3.0",
```

Comment:
`"firefox-client"` is added to the `"devDependencies"` to be able to do an `import type FirefoxClient from 'firefox-client'` across npm versions (it is needed only in the devDependencies because the `import type` statements are removed from the final build).